tests/model_registry/model_catalog/conftest.py

```
@@ -79,3 +92,49 @@ def update_configmap_data_add_model(
+            password=test_idp_user.password,
+        )
+        yield get_openshift_token()
+        LOGGER.info(f"Logging in as {original_user}")
+        login_with_user_password(
+            api_address=api_server_url,
+            user=original_user,
+        )
+    elif user == "sa_user":
+        yield create_inference_token(service_account)
+    else:
+        raise RuntimeError(f"Unknown user type: {user}")
```

Ruff (0.13.0)

Avoid specifying long messages outside the exception class

(TRY003)